### DISCUSSION TITLE
How to disable helpers on babel transform

### DISCUSSION THREAD

**Discussion by nexusprime-here** [babel/babel#17037] [Q&A]
My node is in v22, but babel emit a helper function to async function like "_asyncToGenerator". How to disable it?

**Comment by liuxingbaoyu** [comment]
https://babeljs.io/docs/options#targets
Please try specifying `targets`. :)

**Reply by nexusprime-here** [reply]
Not working 😭

**Reply by nicolo-ribaudo** [reply]
Please share a reproduction
